grails-app/controllers/au/org/ala/spatial/portal/LogController.groovy

```diff
@@ -7,12 +7,14 @@ import org.apache.http.client.methods.HttpGet
 
 class LogController {
     def hubWebService
+    def authService
     static allowedMethods = [index:'POST', search:'GET']
 
     def index() {
         String url = "${grailsApplication.config.layersService.url}/log/"
         def headers = [:]
         headers.put("apiKey",grailsApplication.config.api_key)
```

Comment:
It looks like this is sent to the client (line 16 which you didn't modify). We discussed this a few weeks back - should it be removed?